CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/ActivityExtensions.cs

```diff
@@ -142,6 +147,31 @@ public static class ActivityExtensions
             return activity;
         }
 
+        /// <summary>
+        /// Notifies the mentioned users.
+        /// </summary>
+        /// <param name="replyActivity">The reply activity.</param>
+        /// <returns>Modified activity.</returns>
+        public static Activity NotifyMentionedUsers(this Activity replyActivity)
```

Comment:
Thanks Ramjot! I also chatted w/ Leon about this and tried it on a test bot. Let me know if it turns out otherwise.